xinitrc

```diff
@@ -1,2 +1,2 @@
-
-/home/bin/start-x
+xbindkeys
+qtile start
```

Comment:
Actually calling start-x is preffered if it exists, otherwise we can call xbindkeys and qtile directly. Why are you removing start-x anyway?